release-tools/prow.sh

```diff
@@ -801,11 +801,12 @@ install_snapshot_controller() {
   fi
 
   cnt=0
-  expected_running_pods=$(curl https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/"${CSI_SNAPSHOTTER_VERSION}"/deploy/kubernetes/snapshot-controller/setup-snapshot-controller.yaml | grep replicas | cut -d ':' -f 2-)
-  while [ "$(kubectl get pods -l app=snapshot-controller | grep 'Running' -c)" -lt "$expected_running_pods" ]; do
+  expected_running_pods=$(awk -F '[: ]+' '$1 == "" && $2 == "replicas" {print $3; exit}' "${CONTROLLER_DIR}/deploy/kubernetes/snapshot-controller/setup-snapshot-controller.yaml")
+  expected_namespace=$(awk -F '[: ]+' '$1 == "" && $2 == "namespace" {print $3; exit}' "${CONTROLLER_DIR}/deploy/kubernetes/snapshot-controller/setup-snapshot-controller.yaml")
+  while [ "$(kubectl get pods -n "$expected_namespace" -l app=snapshot-controller | grep 'Running' -c)" -lt "$expected_running_pods" ]; do
     if [ $cnt -gt 30 ]; then
         echo "snapshot-controller pod status:"
-        kubectl describe pods -l app=snapshot-controller
+        kubectl describe pods -n "$expected_namespace" -l app=snapshot-controller
```

Comment:
prow.sh should be changed in csi-release-tools first.

Reply:
Sure, It has been changed already, PR https://github.com/kubernetes-csi/csi-release-tools/pull/121